README.md

```diff
@@ -7,14 +7,28 @@
 
 # Harbor Scanner Adapter for Aqua CSP Scanner
 
-The Harbor [Scanner Adapter][image-vulnerability-scanning-proposal] for Aqua CSP is a service that translates
-the [Harbor][harbor-url] scanning API into `scannercli` commands and allows Harbor to use Aqua CSP scanner
-for providing vulnerability reports on images stored in Harbor registry as part of its vulnerability scan feature.
-
 > **NOTE**: This adapter is only required if you want Harbor to use Aqua CSP for its image scanning feature.
 > If your objective is to use Aqua CSP to provide its own analysis reports against images stored in Harbor,
 > that can be achieved without this adapter.
 
+The Harbor Scanner Adapter for Aqua CSP is a service that translates the Harbor scanning API into `scannercli` commands
+and allows Harbor to use Aqua CSP ad hoc scanning for providing vulnerability reports on images stored in Harbor
+registry as part of its vulnerability scan feature.
+
+The adapter service implements the [Pluggable Scanners API v1.0][harbor-pluggable-scanner-api] and communicates with
+the Aqua server to fulfil the contract required by Harbor scanning API. This contract is based on OS and application
+vulnerabilities, which implies that *Harbor scanning API and Harbor Portal are only capable of handling and displaying
+OS and application vulnerabilities*.
+
+To see the whole report generated by the Aqua scanner, including detected *malware* and *sensitive data*, you have
+to navigate from Harbor Portal to Aqua Management Console. In other words, displaying malware or sensitive data in
+Harbor Portal requires changes to the Pluggable Scanners API, which is owned by the Harbor community and contributed to
+by other security vendors.
+
+Similarly, proprietary *image assurance policies* that can be configured in the Aqua Management Console are not
+integrated with the Harbor deployment security checker through Pluggable Scanners API. Therefore, Aqua assurance
+policies are not merged, displayed nor enforced by Harbor in any way.
```

Comment:
```suggestion
To see the whole report generated by the Aqua scanner, including detected *malware* and *sensitive data*, you have to navigate from Harbor Portal to Aqua Management Console.

Similarly, Harbor does not have support for displaying or enforcing Aqua's *image assurance policies*. You will need to use the Aqua Management Console to view and configure these policies.
```

I think we can skip talking about changes to the Harbor Pluggable Scanners API and just stick to being clear about using the Aqua console for the features that aren't supported in Harbor, wdyt?

Reply:
Yes. I like it. It's more concise this way.